### ISSUE TITLE
Problems typing numbers into generic step scan configuration

### ISSUE THREAD

**Issue by helfrij** [acquaman/acquaman#2188]
Gosia reported that the generic step scan configuration has some spinboxes that make numbers tricky to enter sometimes. This can be seen if trying to change a number like 0.05 to 0.5 by deleting the extra zero. The reported behavior appears to have something to do with a set precision--the spinboxes may not like having values with less decimals than expected. Need to look into it to determine the exact cause.

**Comment by iainworkman** [comment]
It looks as if this was caused by the valueChanged() signal being used on the spin boxes for the first control.

The problem didn't occur on controls for the second control, as it was wired to the editingFinished() signal. The only thing I can think of here is that because the slot to which the valueChanged() signal was wired updated the scan information label, the spinbox in which you are editing momentarily lost focus which caused its value to be validated. So, if you tried to enter ```0.``` without yet adding the values after the decimal point the validation failed and wouldn't let you enter it.

Anyway, I've wired the first set of widgets to the same slot as the second (editingFinished()) and this solves the input annoyance, at the expense of the scan information (number of points, estimated time) only being updated once the spin box loses focus.
